src/controller/userController.js

```diff
@@ -288,3 +288,66 @@ export const userSignin = (req, res, next) => {
       });
     });
 };
+
+/**
+ *
+ * @api {post} /api/v1/users/:userId Login an existing user
```

Comment:
The route as implemented is a GET route " @api {get} /api/v1/users/userId"

Reply:
Updated, thanks.